src/models/tests/reach_curve_test.py

```diff
@@ -44,20 +44,24 @@ def by_spend(self, spend: [float], max_frequency: int = 1) -> ReachPoint:
 class ReachCurveTest(absltest.TestCase):
     def test_reach(self):
         model = FakeReachCurve([ReachPoint([200], (100,))])
+        model._fit()
```

Comment:
Note that FakeReachCurve belongs to ReachCurves which belongs to ReachSurface.
`ReachSurface` has done `_fit()` in `__init__()`.  See line 42 of https://github.com/world-federation-of-advertisers/planning-evaluation-framework/blob/abe1c96ec93cf4d6c2d04808d8211c7c770739d2/src/models/reach_surface.py#L24

So this line is ok but not necessary?

Reply:
Line 42 of `reach_surface.py` is deleted in this PR.  After this PR, it is necessary to always explicitly call `_fit()` after a model is instantiated.